src/components/share/ShareDirective.js

```diff
@@ -21,15 +23,25 @@
               var shortenUrl = scope.options.shortenUrl;
               scope.qrcodegeneratorPath = scope.options.qrcodegeneratorPath;
               scope.mobile = gaBrowserSniffer.mobile;
+              scope.gteIE11 = (gaBrowserSniffer.msie >= 11);
+              scope.gteChrome42 = (gaBrowserSniffer.chrome >= 42);
+              scope.userMessage = '';
 
               if (!gaBrowserSniffer.mobile) {
                 $('.ga-share-icon').tooltip({
                   placement: 'bottom'
                 });
+
+                $('#permalinkInput').tooltip({
+                  placement: 'bottom'
+                });
               }
+
               $('.ga-share-permalink input').on({
                 focus: function() {
-                  this.setSelectionRange(0, 9999);
+                  if (!gaBrowserSniffer.mobile) {
```

Comment:
No focus on mobile ?  in that case you can put this event in the previous `if  (!gaBrowserSniffer.mobile) `